pkg/cluster/database.go

```diff
@@ -329,7 +329,8 @@ func (c *Cluster) execCreateDatabaseSchema(databaseName, schemaName, dbOwner, sc
 	if !c.databaseSchemaNameValid(schemaName) {
 		return nil
 	}
-	c.logger.Infof("%s %q owner %q", doing, schemaName, schemaOwner)
+	//c.logger.Infof("%s %q owner %q", doing, schemaName, schemaOwner)
+	c.logger.Debugf("%s with schemaName:%q schemaOwner:%q dbOwner:%q databaseName:%q statement:%q operation:%q", doing, schemaName, schemaOwner, dbOwner, databaseName, statement , operation)
```

Comment:
can you revert this change? This debug line is not aligned with the rest of the logs

Reply:
reverted as requested.